metadata/README.md

```diff
@@ -24,4 +24,21 @@ erDiagram
         string Metadata_InChI "International Chemical Identifier"
         string Metadata_InChIKey "Hashed InChI"
     }
+    WELL }o--o| ORF : ""
+    ORF {
+        string Metadata_JCP2022 PK "Perturbation identifier"
+        string Metadata_NCBI_Gene_ID "NCBI unique identifier of a gene"
+        string Metadata_broad_sample ""
+        string Metadata_Name ""
+        string Metadata_Vector ""
+        string Metadata_Transcript ""
+        string Metadata_Symbol ""
+        string Metadata_Taxon_ID ""
+        string Metadata_Gene_Description ""
+        string Metadata_Annot_Gene_Symbol ""
+        string Metadata_Annot_Gene_ID ""
+        string Metadata_Prot_Match ""
+        string Metadata_Insert_Length ""
```

Comment:
I can probably guess what these are but I don't know for sure.

Reply:
I think Metadata_Prot_Match = % match to protein sequence

Reply:
`Insert_Length`:

![image](https://user-images.githubusercontent.com/1210428/203372086-aa602605-ef74-4f1f-b124-ee440dca7dfb.png)